### ISSUE TITLE
The training time is slower than centernet, and the training accuracy rate of the same epoch is not as high as centernet.

### ISSUE THREAD

**Issue by enhenghengheng** [ZJULearning/ttfnet#36]
The same learning rate, the same batch_size, but the training time is 30% slower than centernet, and under the same epoch, the correct rate is not as high as centernet. Why?
The configurations used are their default configurations.
